pkgs/applications/virtualization/vmware-workstation/default.nix

```diff
@@ -256,17 +256,15 @@ stdenv.mkDerivation rec {
     cp -r $unpacked/lib $out/lib/vmware/
 
     ## VMware Tools + Virtual Printer
```

Comment:
Just cosmetics, but maybe remove the ` + Virtual Printer` from the comment, since all install code related to printing has been removed?

Reply:
Done.